### PR TITLE
2nd PR for front end code

### PR DESCRIPTION
**Pull request by kumarakash101**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,34 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
+import Header from "./MyComponents/Header";
+import Body from "./MyComponents/Body";
+import Footer from "./MyComponents/Footer";
+import React, { useState } from "react";
+//import { Routes, Route } from "react-router-dom";
```
Comment (juhiaugustine) [inline src/App.js:7]: Can you remove this unused import

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,34 @@
...
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+//import Home from "./MyComponents/Home";
```
Comment (juhiaugustine) [inline src/App.js:9]: Can you remove this import

File: src/MyComponents/Contactus.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+import emailjs from "emailjs-com";
+import React, { useState } from "react";
+
+export default function Contactus() {
+  const [Name, setFirstName] = useState("");
+  const [uemail, setemail] = useState("");
+  const [usubject, setsubject] = useState("");
+  const [umessage, setmessage] = useState("");
+  const [numbers, setnumber] = useState("");
+  //const [emailpattern, setemailpattern] = useState("");
```
Comment (juhiaugustine) [inline src/MyComponents/Contactus.js:11]: Can you remove this unused line

File: src/MyComponents/Contactus.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+  const [error, setError] = useState(false);
+  function sendemail(e) {
+    let numbers = /^[0-9]+$/;
+    //let emailpattern = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```
Comment (juhiaugustine) [inline src/MyComponents/Contactus.js:16]: Can you remove this unused line

**Review by juhiaugustine — APPROVED** [review]

File: src/MyComponents/Contactus.js (hunk 4 above)
Comment (kumarakash101) [reply src/MyComponents/Contactus.js:16]: This has been done

File: src/MyComponents/Contactus.js (hunk 3 above)
Comment (kumarakash101) [reply src/MyComponents/Contactus.js:11]: This has been done

File: src/App.js (hunk 2 above)
Comment (kumarakash101) [reply src/App.js:9]: This has been done

File: src/App.js (hunk 1 above)
Comment (kumarakash101) [reply src/App.js:7]: This has been done

**Pull request merged**